mountpoint-s3-client/examples/client_benchmark.rs

```diff
@@ -22,8 +22,8 @@ fn init_tracing_subscriber() {
 
     let subscriber = Subscriber::builder()
         .with_env_filter(EnvFilter::from_default_env())
+        .with_ansi(supports_color::on(supports_color::Stream::Stderr).is_some())
```

Comment:
This is lifted from how we configure it for Mountpoint proper: https://github.com/awslabs/mountpoint-s3/blob/c336f951a4934724f2975f76df24e22c0d299afc/mountpoint-s3-fs/src/logging.rs#L161